form/events.rst

```diff
@@ -219,7 +215,8 @@ View data        Normalized data transformed using a view transformer
 
 .. caution::
 
-    At this point, you cannot add or remove fields to the form.
```

Comment:
Same here.

Reply:
Here I just make explicit that child fields cannot be changed on `POST_SUBMIT`, is that wrong?

Reply:
Sorry, misread the changes. This is correct.